applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_transonic_perturbation_potential_element.cpp

```diff
@@ -55,7 +55,18 @@ void GenerateTransonicPerturbationElement(ModelPart& rModelPart) {
 }
 
 void GenerateTransonicPerturbationUpwindElement(ModelPart& rModelPart) {
+    // Variables addition
+    // Set the element properties
     Properties::Pointer pElemProp = rModelPart.CreateNewProperties(1);
+    rModelPart.GetProcessInfo()[FREE_STREAM_DENSITY] = 1.225;
+    rModelPart.GetProcessInfo()[FREE_STREAM_MACH] = 0.6;
+    rModelPart.GetProcessInfo()[HEAT_CAPACITY_RATIO] = 1.4;
+    rModelPart.GetProcessInfo()[SOUND_VELOCITY] = 340.3;
+    rModelPart.GetProcessInfo()[MACH_LIMIT] = 0.94;
+    rModelPart.GetProcessInfo()[MACH_SQUARED_LIMIT] = 3.0;
+    rModelPart.GetProcessInfo()[CRITICAL_MACH] = 0.99;
+    rModelPart.GetProcessInfo()[UPWIND_FACTOR_CONSTANT] = 1.0;
```

Comment:
This is not necessary because the parameters are already specified when generating the current element.